src/web_ui/src/components/common/ProductSearch/index.test.js

```diff
@@ -6,14 +6,13 @@ import { act } from "react-dom/test-utils";
 import ProductSearch from "./index";
 
 let container = null;
+
 beforeEach(() => {
-  // setup a DOM element as a render target
   container = document.createElement("div");
   document.body.appendChild(container);
 });
 
 afterEach(() => {
-  // cleanup on exiting
```

Comment:
leave comments its not acceptable to remove components that define component function

Reply:
added the comment back